elliottlib/cli/find_builds_cli.py

```diff
@@ -228,7 +226,7 @@ def _attached_to_open_erratum_with_correct_pv(kind, results, errata):
                 if errata_version_cache[e] == get_release_version(b.product_version):
                     same_version_exist = True
                     break
-        if not same_version_exist or not b.attached_to_open_erratum:
+        if (not same_version_exist or not b.attached_to_open_erratum) and not b.attached_to_shipped_erratum:
```

Comment:
according to De Morgan's laws, this could be reduced to 
```
if not (same_version_exist and b.attached_to_open_erratum or b.attached_to_shipped_erratum):
```
but for readability maybe like
```
if not ( same_version_exist and b.attached_to_open_erratum) and not b.attached_to_shipped_erratum:
```

Reply:
I prefer the first one `if not (same_version_exist and b.attached_to_open_erratum or b.attached_to_shipped_erratum):`

Reply:
Any could go for me

Reply:
Updated.

Reply:
can i suggest that the same treatment above for `if b.attached_to_open_erratum:` be applied also `if b.attached_to_shipped_erratum:` and feed into `same_version_exist` (perhaps rename that as `in_same_version`)?

then i think this becomes:
```suggestion
        if not in_same_version:
```
... because the only way that can ever be set is if it was already attached to an advisory for the same version.